build-selftests/build_selftests.sh

```diff
@@ -10,9 +10,9 @@ KERNEL="$1"
 TOOLCHAIN="$2"
 export KBUILD_OUTPUT="$3"
 
-LLVM_VER="$(llvm_version $TOOLCHAIN)" && :
-if [ $? -eq 0 ]; then
-	export LLVM="-$LLVM_VER"
+if [[ $TOOLCHAIN = llvm* ]]; then
```

Comment:
same as above.

Reply:
We have this logic in 3 different paces (and should probably move this into a function at a later stage), but this is slightly different:
`if [[ $TOOLCHAIN = llvm* ]]; then`
vs
`if [ $TOOLCHAIN = "llvm" ]; then`

is this expected? I would assume this is because the way we call the action is different. Is there any ways to uniformize it?

Reply:
oops, it should all be ` $TOOLCHAIN = "llvm" `.